src/data/posts/postsData.ts

```diff
@@ -1,31 +1,33 @@
 import PostType, {FoodGroup} from "../../modules/postType";
-
 const posts : Array<PostType> = [
   {
     "_id": 10,
-    "recipe_id": 715467,
-    "user_id": 1,
-    "caption": "10/10 would make again!",
+    "recipeId": "643b7f955da0d4402e90938e",
+    "userId": "1",
```

Comment:
should we add more variance to these posts than the userId? or is the purpose of this to repost the same recipe to the same group multiple times and illustrate the post:recipe ratio can be something other than 1:1?

Reply:
These were just used to test post rendering, they will be deprecated eventually.